browser/src/ErrorBoundary.tsx

```diff
@@ -57,14 +63,45 @@ Browser: ${navigator.userAgent}
           <DocumentTitle title="Error" />
           <PageHeading>Something Went Wrong</PageHeading>
           <p>An error prevented this page from being displayed.</p>
+          <p>This is a bug.</p>
+          <p>
+            Please describe what you did to cause this page crash
```

Comment:
Let's soften this wording a bit, even sophisticated users like our audience can get defensive about their computer skills and to be fair, if the page is crashing, that's probably something _we_ did to cause it.

How about "Please describe what you were trying to do at the time the page crashed"?

Reply:
Aha, yep -- very good call.